get_last_revision_titles_en.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+"""
+    get_last_revision_en.py
```

Comment:
As there is just a change in the endpoint in this example, this is not adding any value and we don't need this file.